### PR TITLE
Don't cache dynamic values to smartapi_specs.json

### PR DESCRIPTION
**Pull request by tokebe**
*(addresses biothings/smartapi-kg.js#36)*

This change cleans a few dynamic values before caching `smartapi_specs.json`, and updates the default cached files.

### CODE REVIEW

File: src/controllers/cron/update_local_smartapi.js [hunk 1]
```diff
@@ -173,7 +173,7 @@ const updateSmartAPISpecs = async () => {
     });
 }
 module.exports = () => {
-    cron.schedule('*/10 * * * *', async () => {
+    cron.schedule('* * * * *', async () => {
```
Comment (newgene) [inline src/controllers/cron/update_local_smartapi.js]: @tokebe this now runs every minute?

Comment (tokebe) [reply src/controllers/cron/update_local_smartapi.js]: Ah, I made that change during testing and forgot to remove it. Commit fixing this incoming.

**Comment by newgene** [comment]
@tokebe If the cached JSON structure changes, any corresponding change is needed where these cached files are consumed (like `smartapi-kg` or other packages)?

**Comment by tokebe** [comment]
I avoided changing the cached structure in a way where any other changes would be necessary. The only case where a problem should be able to arise is if the structure from the newly retrieved specs was different.

Original structure:
```
{
  "took": number,
  "total": number,
  "max_score": number,
  "hits": [actual data we want]
}
```
After this change:
```
{
  "hits": [actual data we want]
}
```
Nothing inside of `hits` is modified, and it should still be accessed in exactly the same way after being read from the cache (e.g., `cachedFile.hits`).

**Comment by newgene** [comment]
Got it, perfect! Merging this PR now...

**Comment by tokebe** [comment]
For ease of merge conflict resolution, the two data files from this PR may be ignored -- they were just produced by running the job with the changes to the caching code.

**Comment by newgene** [comment]
Two files under `data` have conflicts, which prevents me from merging right now. No hurry, you can resolve these conflicts when you have chance (can exclude these two files from this PR too)

**Comment by tokebe** [comment]
@newgene Conflicts removed, branch is ready for merge.

**Pull request merged**